tidb/assets/dashboards/one-screen-overview.json

```diff
@@ -1 +0,0 @@
,"display_type":"line"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]}},{"id":5486782423063084,"definition":{"title":"AVG of Parse Duration","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"q":"per_second(avg:tidb_cluster.tidb.session_parse_duration_seconds.sum{type:wait} by {host}), per_second(avg:tidb_cluster.tidb.session_parse_duration_seconds.count{type:wait} by {host}), per_second(avg:tidb_cluster.tidb.session_parse_duration_seconds.sum{type:wait} by {host})/per_second(avg:tidb_cluster.tidb.session_parse_duration_seconds.count{type:wait} by {host})","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]}},{"id":405411499029966,"definition":{"title":"AVG of PD TSO RPC Duration","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"q":"per_second(avg:tidb_cluster.tidb.pd_client_request_handle_requests_duration.sum{type:wait} by {host}), per_second(avg:tidb_cluster.tidb.pd_client_request_handle_requests_duration.count{type:wait} by {host}), per_second(avg:tidb_cluster.tidb.pd_client_request_handle_requests_duration.sum{type:wait} by {host})/per_second(avg:tidb_cluster.tidb.pd_client_request_handle_requests_duration.count{type:wait} by {host})","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]}},{"id":7739111915948736,"definition":{"title":"AVG of Compile Duration","title_size":"16","title_align":"left","show_legend":true,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"time":{},"type":"timeseries","requests":[{"q":"per_second(avg:tidb_cluster.tidb.session_compile_duration_seconds.sum{type:wait} by {host}), per_second(avg:tidb_cluster.tidb.session_compile_duration_seconds.count{type:wait} by {host}), per_second(avg:tidb_cluster.tidb.session_compile_duration_seconds.sum{type:wait} by {host})/per_second(avg:tidb_cluster.tidb.session_compile_duration_seconds.count{type:wait} by {host})","style":{"palette":"dog_classic","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"scale":"linear","label":"","include_zero":true,"min":"auto","max":"auto"},"markers":[]}}]}},{"id":4061043905521020,"definition":{"title":"TiKV","type":"group","layout_type":"ordered","widgets":[]}},{"id":469462288990948,"definition":{"title":"PD","type":"group","layout_type":"ordered","widgets":[]}}],"template_variables":[{"name":"cluster","default":"*","prefix":"cluster"}],"layout_type":"ordered","is_read_only":false,"notify_list":[],"id":"8ci-dz8-ugh"}
```

Comment:
Is this PR removing this dashboard?

Reply:
Yes, I tried to remove it but CI checks failed.

Reply:
I try to keep one dashboard, whose title is `TiDB Overview`, and remove others.

Reply:
We avoid deleting dashboards, do you have any additional context on why this dashboard should be deleted?

Reply:
Previously, TiDB Integration provided two dashboards, a "cluster overview" dashboard and a supplement to the "cluster overview".

In this new version we have simplified the metrics. with new metrics, the dashboard can be optimized to one. If we can't delete the dashboard, could you give us a better solution? Thanks.

Reply:
It's possible to delete the dashboard, but it's a bit tricky on our side so I prefer to make sure we do that properly for good reasons 😄 
You can remove the dashboard following this https://github.com/DataDog/integrations-extras/pull/1094#discussion_r765553318